test/frontend-integration-test/README.md

```diff
@@ -0,0 +1,24 @@
+# Frontend integration test
```

Comment:
Thank you for adding the additional running mode, and the documenting the usage.

/cc @jlyaoyuli in case frontend team has a similar or better way to run FE integration test in custom deployment.